src/components/Common/ButtonDelete.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react'
```

Comment:
Same here. I think I forgot to save the message

```javascript
import React from 'react'

export function ButtonDelete({ onClick, extraClassName = '' }) {
  return <div onClick={onClick} className={`sw-ButtonDelete ${extraClassName}`}
}
```

Reply:
> @gabitoesmiapodo Custom Gas Price arrows always change the decimals to `.1`
> 
> ![custom-gas-price_always01](https://user-images.githubusercontent.com/3315606/47096786-4e4b6200-d206-11e8-909d-296c5cca6477.gif)

@fernandomg 

Ugh, this is a sh*tty one, let me see if I can make it work.